test/e2e/testkit/mocks/backend_deployment.go

```diff
@@ -34,6 +36,97 @@ func NewBackendDeployment(name, namespace, configmapName, dataPath string) *Back
 	}
 }
 
+func NewLogBackendDeployment(name, namespace, configmapName, dataPath, fluentdConfigMap string) *BackendDeployment {
+	return &BackendDeployment{
+		name:              name,
+		namespace:         namespace,
+		configmapName:     configmapName,
+		dataPath:          dataPath,
+		fluentdConfigName: fluentdConfigMap,
+	}
+}
+
+func (d *BackendDeployment) K8sObjectWithFluentD(labelOpts ...testkit.OptFunc) *appsv1.Deployment {
```

Comment:
Do we have a use-case to deploy the collector without FluentD container?  Maybe override the existing `K8sObject` function with this body.

Reply:
K8sObject used for another tests, this one will create a different deployment form others